### ISSUE TITLE
Missing SoftMax function

### ISSUE THREAD

**Issue by farzadsaffari** [eeyhsong/EEG-Conformer#10]
Hi

Thanks for sharing your implementation. Really appreciate it. 
Even though it is stated that a SoftMax activation function has been sued in the classification part, in your implementation you have not used this function in the last layer of your model. 
could you clarify whether there is a reason or if I'm just missing something? 
Thanks.

**Comment by eeyhsong** [comment]
hello, thanks for your intersest!
The use of [torch.nn.CrossEntropyLoss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html) said this case is equivalent to the combination of [LogSoftmax](https://pytorch.org/docs/stable/generated/torch.nn.LogSoftmax.html#torch.nn.LogSoftmax) and [NLLLoss](https://pytorch.org/docs/stable/generated/torch.nn.NLLLoss.html#torch.nn.NLLLoss).
So we use this function in the code.

**Comment by farzadsaffari** [comment]
Ok yes, that work as well. 
Thanks.
